src/main/java/com/openlattice/datastore/permissions/controllers/PermissionsController.java

```diff
@@ -248,11 +262,33 @@ public AuthorizationManager getAuthorizationManager() {
         return authorizations;
     }
 
+    @NotNull
+    @Override
+    public AuditingManager getAuditingManager() {
+        return auditingManager;
+    }
+
     private List<Acl> getOrganizationExternalDatabaseAcls( List<Acl> acls ) {
-        Set<AclKey> aclKeys = acls.stream().map(acl -> new AclKey( acl.getAclKey() )).collect( Collectors.toSet() );
+        Set<AclKey> aclKeys = acls.stream().map( acl -> new AclKey( acl.getAclKey() ) ).collect( Collectors.toSet() );
         Set<AclKey> allOrgExternalDBAclKeys = securableObjectResolveTypeService
                 .getOrganizationExternalDatabaseAclKeys( aclKeys );
         return acls.stream().filter( acl -> allOrgExternalDBAclKeys.contains( acl.getAclKey() ) )
                 .collect( Collectors.toList() );
     }
+
+    private List<AuditableEvent> createAuditableEvents( List<Acl> acls, AuditEventType eventType ) {
+        return acls.stream().map( acl ->
```

Comment:
~~I think in this case (`acls` being a concrete List and never a lazy list/streamed collection), prefer using `.map {}` over `.stream().map(`~~

Okay so nevermind b/c this is java but I think the above is true in kotlin in general